### PR TITLE
Resize event function

### PR DESCRIPTION
**Pull request by LaineZ**
Added resize event processing with:
```
engine.get_resize() -> Option<(u16, u16)>
```
where first argument is terminal width; second - terminal height

### CODE REVIEW

**Comment by VincentFoulon80** [comment]
Thank you for the contribution ! Everything's looking good ! Also thanks for the code cleaning 👍 
Maybe we can write a little example for the documentation, and that'll be perfect !

File: src/lib.rs [hunk 1]
```diff
@@ -728,6 +741,16 @@ impl ConsoleEngine {
         None
     }
 
+    /// Give the terminal resize event
```
Comment (VincentFoulon80) [inline src/lib.rs:744]:
```suggestion
    /// Give the terminal resize event
    ///
    /// usage:
    /// ```
    /// if let Some(width, height) = engine.get_resize() {
    ///     // do something
    /// }
```

**Review by VincentFoulon80 — COMMENTED** [review]
Here is an example. I didn't tested the code though.

File: src/lib.rs (hunk 1 above)
Comment (LaineZ) [reply src/lib.rs:744]: Yes, adding a example is good idea, but example snippet contains error

Here a right version
```suggestion
    /// Give the terminal resize event
    ///
    /// usage:
    /// ```
    /// if let Some((width, height)) = engine.get_resize() {
    ///     // do something
    /// }
    /// ```
```

**Comment by VincentFoulon80** [comment]
Thank you @LaineZ !
Release target : 1.3.0

**Pull request merged**